### PR TITLE
refactor: Changing Table Name

### LINKED ISSUE #6389: Rename association table to a proper name
There's a table called association

Can anyone guess what it is used for?

That's right. Linking ticket tags and tickets. So obvious...

### PR DESCRIPTION
**Pull request by kushthedude**
Fixes #6389

### CODE REVIEW

**Comment by kushthedude** [comment]
@iamareebjamal There are some unwanted changes I think generated by Alembic, Should I remove them or are they required?

**Comment by iamareebjamal** [comment]
There are no unwanted changes in alembic, the entire migration file generated is unwanted. It is creating a new table because obviously alembic cannot detect if you want to rename a table or create one

**Comment by kushthedude** [comment]
> There are no unwanted changes in the alembic, the entire migration file generated is unwanted. It is creating a new table because obviously, alembic cannot detect if you want to rename a table or create one

Is it fine now?

**Comment by kushthedude** [comment]
@iamareebjamal Please Review

File: migrations/versions/change_table_name_asso.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+
+ # revision identifiers, used by Alembic.
+ # This is a hand-written migration , Please check thouroughly .
+revision = '90d62fe3b5f4'
+down_revision = '90d62fe3b5e3'
+
+
+def upgrade():
+	op.rename('ticket_tagging', 'association')
```
Comment (iamareebjamal) [inline migrations/versions/change_table_name_asso.py]: Wrong

**Comment by iamareebjamal** [comment]
Please read once before asking for reviews

File: migrations/versions/change_table_name_asso.py (hunk 1 above)
Comment (kushthedude) [reply migrations/versions/change_table_name_asso.py]: Do I need to use 
```py
op.execute('`psql Query for Changing Table Name`')
```
@iamareebjamal

Comment (iamareebjamal) [reply migrations/versions/change_table_name_asso.py]: `op.rename('ticket_tagging', 'association')`

Read this again. Slowly...

**Comment by prateekj117** [comment]
@kushthedude Please update the PR or close it, I will make a new one. Hardly a one day job.

**Pull request closed without merging**